crates/oxc_codegen/tests/integration/esbuild.rs

```diff
@@ -8,17 +8,18 @@ use crate::tester::{test, test_minify};
 #[test]
 fn test_number() {
     // Check "1eN"
-    test("x = 1e-100", "x = 1e-100;\n");
-    test("x = 1e-4", "x = 1e-4;\n");
-    test("x = 1e-3", "x = 1e-3;\n");
-    test("x = 1e-2", "x = 1e-2;\n");
-    test("x = 1e-1", "x = 1e-1;\n");
-    test("x = 1e0", "x = 1e0;\n");
-    test("x = 1e1", "x = 1e1;\n");
-    test("x = 1e2", "x = 1e2;\n");
-    test("x = 1e3", "x = 1e3;\n");
-    test("x = 1e4", "x = 1e4;\n");
-    test("x = 1e100", "x = 1e100;\n");
+    // TODO FIXME
```

Comment:
I'm undecided whether we should print numbers differently in `--minify-whitespace` vs normal mode.